xml/System/Byte.xml

```diff
@@ -1,6995 +1,6995 @@
-      <MemberSignature Language="ILAsm" Value=".method static hidebysig bool System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryReadLittleEndian(valuetype System.ReadOnlySpan`1&lt;unsigned int8&gt; source, bool isUnsigned, [out] unsigned int8&amp; value) cil managed" />
-      <MemberSignature Language="DocId" Value="M:System.Byte.System#Numerics#IBinaryInteger&lt;System#Byte&gt;#TryReadLittleEndian(System.ReadOnlySpan{System.Byte},System.Boolean,System.Byte@)" />
-      <MemberSignature Language="VB.NET" Value=" Shared Function TryReadLittleEndian (source As ReadOnlySpan(Of Byte), isUnsigned As Boolean, ByRef value As Byte) As Boolean Implements IBinaryInteger(Of Byte).TryReadLittleEndian" />
-      <MemberSignature Language="F#" Value="static member System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryReadLittleEndian : ReadOnlySpan&lt;byte&gt; * bool * byte -&gt; bool" Usage="System.byte.System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryReadLittleEndian (source, isUnsigned, value)" />
-      <MemberSignature Language="C++ CLI" Value=" static bool System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryReadLittleEndian(ReadOnlySpan&lt;System::Byte&gt; source, bool isUnsigned, [Runtime::InteropServices::Out] System::Byte % value) = System::Numerics::IBinaryInteger&lt;System::Byte&gt;::TryReadLittleEndian;" />
-      <MemberType>Method</MemberType>
-      <AssemblyInfo>
-        <AssemblyName>System.Runtime</AssemblyName>
-        <AssemblyVersion>7.0.0.0</AssemblyVersion>
-        <AssemblyVersion>8.0.0.0</AssemblyVersion>
-      </AssemblyInfo>
-      <AssemblyInfo>
-        <AssemblyName>mscorlib</AssemblyName>
-      </AssemblyInfo>
-      <AssemblyInfo>
-        <AssemblyName>netstandard</AssemblyName>
-      </AssemblyInfo>
-      <ReturnValue>
-        <ReturnType>System.Boolean</ReturnType>
-      </ReturnValue>
-      <Parameters>
-        <Parameter Name="source" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="0" FrameworkAlternate="net-7.0;net-8.0" />
-        <Parameter Name="isUnsigned" Type="System.Boolean" Index="1" FrameworkAlternate="net-7.0;net-8.0" />
-        <Parameter Name="value" Type="System.Byte" RefType="out" Index="2" FrameworkAlternate="net-7.0;net-8.0" />
-      </Parameters>
-      <Docs>
-        <param name="source">To be added.</param>
-        <param name="isUnsigned">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
-      </Docs>
-    </Member>
-    <Member MemberName="System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryWriteBigEndian">
-      <MemberSignature Language="C#" Value="bool IBinaryInteger&lt;byte&gt;.TryWriteBigEndian (Span&lt;byte&gt; destination, out int bytesWritten);" />
-      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance bool System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryWriteBigEndian(valuet
```

Comment:
@carlossanlop I think these funny code tags are coming from inheritdoc.
Also, why don't we just import the `<inheritdoc>` tag instead of trying to implement the inheritdoc logic ourselves?

Reply:
It should have for some cases. See for example this PR: https://github.com/dotnet/dotnet-api-docs/pull/9198

I would have to investigate why some inheritdocs are not getting added.

Reply:
From what I can tell from the [inheritdoc proposal in dotnet/csharplang](https://github.com/dotnet/csharplang/pull/1749/files#r275951977), the compiler can either just copy over the inheritdoc tag, or it can expand the documentation. I'm not sure when it does which. But I don't think your tool needs to be in the business of expanding the tag.

Reply:
OK I think I know what could've happened: The APIs already had some documentation, and unless everything says `To be added.`, the porting tool won't insert the `<inheritdoc/>` tag and will instead copy the docs item by item. I did this because the tool assumes that the API owner decided to override/customize the docs instead of inheriting them.